app/models/departure_fetcher.rb

```diff
@@ -1,16 +1,21 @@
+require 'duration'
+
 class DepartureFetcher
   include ActiveModel::SerializerSupport
 
   def initialize(params)
     @time = params[:time] ? Time.zone.parse(params[:time]) : Time.current
     @stop_id = params[:stop_id]
+    @time_limit = params.fetch(:time_limit, 10)
```

Comment:
Do we need to `to_i` this? params that come in via the controller will be strings.